_inc/client/state/settings/actions.js

```diff
@@ -82,32 +83,17 @@ export const updateSetting = ( updatedOption ) => {
 	};
 };
 
-export const updateSettings = ( newOptionValues, type = '' ) => {
+export const updateSettings = ( newOptionValues, noticeMessages = {} ) => {
 	return ( dispatch ) => {
-		let messages = {
```

Comment:
Comment for the reviewer. Basically this block was moved into `mapUpdateSettingsResponseFromApi` and the messages have been delegated to the component dispatching the action to one of the actions creators mapped into its props.